src/clj/game/cards/events.clj

```diff
@@ -176,7 +177,7 @@
                                          (fn [state side card]
                                            (if (= (:cid card) (:cid target))
                                              ((constantly false)
-                                               (toast state :corp "Cannot rez the rest of this turn due to Careful Planning"))
+                                              (toast state :corp "Cannot rez the rest of this turn due to Careful Planning"))
```

Comment:
Hmm I'm not sure about this change... `(constantly false)` is the function and the rest is the arguments which I would expect to be indented a bit... (there is a similar change in agendas).